ktlint-reporter-json/src/test/kotlin/com/pinterest/ktlint/reporter/json/JsonReporterTest.kt

```diff
@@ -67,37 +67,40 @@ class JsonReporterTest {
         reporter.afterAll()
         assertThat(String(out.toByteArray())).isEqualTo(
             """
-[
-	{
-		"file": "/one-fixed-and-one-not.kt",
-		"errors": [
-			{
-				"line": 1,
-				"column": 1,
-				"message": "<\"&'>",
-				"rule": "rule-1"
-			}
-		]
-	},
-	{
-		"file": "/two-not-fixed.kt",
-		"errors": [
-			{
-				"line": 1,
-				"column": 10,
-				"message": "I thought I would again",
-				"rule": "rule-1"
-			},
-			{
-				"line": 2,
-				"column": 20,
-				"message": "A single thin straight line",
-				"rule": "rule-2"
-			}
-		]
-	}
-]
-""".trimStart().replace("\n", System.lineSeparator())
+            [
```

Comment:
I think it's totally unreadable now, can we do something about that? is this how the indent rule works now?

Reply:
No, luckily this is not how the rule works. 

After I aligned the closing quotes with the start quotes,  all indentation tabs in the file were automatically replaced with spaces. That was unexpected and confusing. It also difficult to see in the test results what was wrong.

Instead of "{TAB}" we can also use "\t" if you prefer that.

Reply:
I'd like to not have any of this tbh... Does JsonReport emit tabs instead of spaces? I'd rather change it there to emit spaces, same for CheckstyleReporter

Reply:
Yeah sure, no problem to replace the tab indentation with space indentation.